pkg/util/docker/docker_util.go

```diff
@@ -27,8 +27,8 @@ import (
 )
 
 const (
-	pauseContainerGCR       string = "image:gcr.io/google_containers/pause.*"
-	pauseContainerOpenshift string = "image:openshift/origin-pod"
+	pauseContainerGCR       = "image:.*gcr.io/.*pause.*"
```

Comment:
It might be a bit too vague, as GCR can be used as a private registry, I fear we could match legit container like `gcr.io/random-project/superpause:1.0`